wagtailmenus/modeladmin.py

```diff
@@ -17,11 +17,6 @@ class MainMenuAdmin(ModelAdmin):
     edit_view_class = views.MainMenuEditView
     add_to_settings_menu = True
 
-    def get_form_view_extra_css(self):
```

Comment:
These styles don't seem to be needed anymore. The wagtail UI deals with them perfectly. Also removed the second instance in this file.